### ISSUE TITLE
I want nopondering options

### ISSUE THREAD

**Issue by sinhayday** [featurecat/lizzie#309]
Hi
I want nopondering options.
Yes.  lizzie already has no pondering (space bar toggle)
but when i push spacebar, it stops all search.
i think it's better that when user push spacebar,it stops next pondering not this move's search.

**Comment by featurecat** [comment]
I don't see hour point. we currently keep the variations visible, right?

parmuzinAlexander: what is the matter...

**Comment by ParmuzinAlexander** [comment]
Offer to change what works fine. And what is better not to explain. Need to wait for the next move to stop pondering? It ruin program for me, so I'm worried.

**Comment by featurecat** [comment]
don't worry I'll try not to ruin the program...

**Comment by sinhayday** [comment]
hi parmuzinalexander and featurecat,  im not good at english , so I couldn't clarify what i want to say.
just forget about this issue ^^ thanks to developers.

**Comment by featurecat** [comment]
oh please don't worry. I'm sure you will find a way to say it! Go players can achieve anything 😊. I hope to hear about your idea some day. Thanks

**Comment by Gurxtav** [comment]
I think (just my interpretation/guess) what was meant was this: after I start a game with Leelaz using n button in Lizze, I want an option (space) to make it not think on my turn but only when it is Leela's move (thinking on opponents turn is called pondering). 

I think this is a good idea if you want to think a lot about the game and take lots of thinking time but you don't want your fans to spin all the time.

**Comment by OlivierBlanvillain** [comment]
Can't you simply add `--noponder` to the leelaz options?

**Comment by alreadydone** [comment]
I think --noponder has no effect and you just need Lizzie not to send lz-analyze command to leelaz when it's black or white (whatever your opponent's color is)'s turn. But currently you can just press space every time you press Enter to play leelaz's top choice (to stop analyzing), and also press space every time you make your move (to start analyzing), I think.

**Comment by featurecat** [comment]
correct. we don't handle games in the normal way. it would be nice to be able to let the pondering end. or just to handle the game like a normal gtp interface.

**Comment by OlivierBlanvillain** [comment]
I see, I think normal gtp would make more sense for the "play vs AI" mode since the `--noponder` option would be taken into account automatically.

In my opinion the "play vs AI" mode should be removed in favor of two in game options:

- hide/show move suggestions for white
- hide/show move suggestions for black

Turning both on would result in the current play vs AI mode, but I assume than in the review it my be interesting to turn only one player of to "play a variation against the AI" while still seeing what the AI is thinking during it's turn.

**Comment by alreadydone** [comment]
Hiding suggestions (separately for black or white) won't stop fans from spinning all the time, but is implemented in mylizzie (forgot about Lizzie).

**Comment by OlivierBlanvillain** [comment]
> Hiding suggestions (separately for black or white) won't stop fans from spinning all the time

It should (I mean, we should implement it). I think the result would be a better user experience than the current play vs AI mode.

**Comment by featurecat** [comment]
we should have an equivalent of "new game" that sets the board to the desired handicap, and maybe an additional way to play the AI from the current position.
